### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stas-fomenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
 # splitter-component
-Splitter component for the elastic.io platform
+Splitter is the basic component for the [elastic.io platform](http://www.elastic.io).
 
-## Environment variables
+## Description
+The Splitter processes income messages containing multiple elements that might have to be processed in different ways. The Splitter emmits out the composite message into individual messages, each containing data related to one item.
 
+### How works
+???
```
Comment (A3a3e1) [inline README.md]: Should it be appended here?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,21 @@
 # splitter-component
-Splitter component for the elastic.io platform
+Splitter is the basic component for the [elastic.io platform](http://www.elastic.io).
 
-## Environment variables
+## Description
+The Splitter processes income messages containing multiple elements that might have to be processed in different ways. The Splitter emmits out the composite message into individual messages, each containing data related to one item.
```
Comment (A3a3e1) [inline README.md]: 'emmits' -> 'emits'

File: README.md [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
 
+### How works
+???
+
+#### Environment variables 
 Component does not has any required environment variables, but we suggest to use `EIO_REQUIRED_RAM_MB` in order to avoid `Component run out of memory and terminated` error, recommended value of allocated memory is `512` MB.
```
Comment (A3a3e1) [inline README.md]: `does not have`

File: README.md [hunk 4]
```diff
@@ -1,16 +1,21 @@
...
 
 ## Actions
+### Split Message By Array
+Splits a message into multiple messages using a given separator. The separator is treated as a path to a property inside the message. A message is split when a property is an array and emitted are multiple messages. Otherwise the original message is emitted.
 
-### Split
+For example, we have message:
```
Comment (A3a3e1) [inline README.md]: For example, we have a message

File: README.md (hunk 2 above)
Comment (stas-fomenko) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (stas-fomenko) [reply README.md]: removed

File: README.md (hunk 3 above)
Comment (stas-fomenko) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (stas-fomenko) [reply README.md]: fixed

File: README.md [hunk 5]
```diff
@@ -33,8 +36,14 @@ Our splitting expression is "users". As the output of the component we'll have t
...
+
+- *When splitting expression refers to an object splitter returns this object;*
+- *When splitting expression contains primitive value like ```users:"John"``` or array of primitives like ```users:["John", "Mike", "Anna"]``` splitter emits error.*
+
+#### List of Expected Config fields
+```Split Property``` - use this field to choose a separator.
 
-If splitting expression refers to object splitter just return this object.
+## Known limitations (common for the component)
+No.
```
Comment (A3a3e1) [inline README.md]: ## Documentation links
More information and some examples can be found here: [Splitter documentation](https://www.elastic.io/connectors/splitter-integration/)
and here: [Array splitting and JSONata mapper](https://support.elastic.io/support/solutions/articles/14000069604-array-splitting-and-jsonata-mapper)

**Pull request merged**